src/Query/Scopes/Filters/Fields/Entries.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Statamic\Query\Scopes\Filters\Fields;
+
+use Statamic\Facades;
+use Statamic\Support\Arr;
+use Statamic\Support\Str;
+
+class Entries extends FieldtypeFilter
+{
+    public function fieldItems()
+    {
+        return [
+            'field' => [
+                'type' => 'select',
+                'options' => [
+                    'id' => __('ID'),
+                    'title' => __('Title'),
+                ],
+                'placeholder' => __('Field'),
+                'default' => 'id',
+            ],
+            'operator' => [
+                'type' => 'select',
+                'placeholder' => __('Select Operator'),
+                'options' => [
+                    'like' => __('Contains'),
+                    '=' => __('Is'),
+                    '!=' => __('Isn\'t'),
+                ],
+                'default' => 'like',
+            ],
+            'value' => [
+                'type' => 'text',
+                'placeholder' => __('Value'),
+                'if' => [
+                    'operator' => 'not empty',
+                ],
+            ],
+        ];
+    }
+
+    public function apply($query, $handle, $values)
+    {
+        if ($values['field'] == 'id') {
+            $query->where($handle, $values['operator'], $values['value']);
+
+            return;
+        }
+
+        $config = $this->fieldtype->field()->config();
+
+        $operator = $values['operator'];
+        $value = $values['value'];
+
+        if ($operator === 'like') {
+            $value = Str::ensureLeft($value, '%');
+            $value = Str::ensureRight($value, '%');
+        }
+
+        $ids = Facades\Entry::query()
+            ->when($config['collections'], fn ($query) => $query->whereIn('collection', $config['collections']))
+            ->where($values['field'], $operator, $value)
+            ->get(['id'])
+            ->map(fn ($entry) => $entry->id())
+            ->all();
+
+        if (($config['max_items'] ?? 0) == 1) {
+            $query->whereIn($handle, $ids);
+
+            return;
+        }
+
+        if (empty($ids)) {
+            $query->where($handle, -1);
```

Comment:
When `$ids` is empty, I just get an error.
`strtolower(): Argument #1 ($string) must be of type string, array given`

I'm guessing that the `where($handle, -1)` is just trying to trick the query into returning no results?

Reply:
Yep that was the idea.

Reply:
I've tidied that up now - it needed changed to whereJsonContains.